app.rb

```diff
@@ -0,0 +1,202 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'person'
+require_relative 'rental'
+
+class App
```

Comment:
- [ ] [OPTIONAL] Great implementation @yohannesdagnachew , however, for a better user experience (UX) you can consider displaying a goodbye message after a user exits the app.